prow/config.yaml

```diff
@@ -197,34 +197,39 @@ presubmits:
       - name: kubeconfig
         secret:
           secretName: istio-e2e-rbac-kubeconfig
-    run_after_success:
-    - name: istio-unit-test
-      agent: kubernetes
-      context: prow/istio-unit-test.sh
-      rerun_command: "/test istio-unit-test"
-      trigger: "(?m)^/test (istio-unit-test)?,?(\\s+|$)"
-      max_concurrency: 8
-      spec:
-        containers:
-        - image: gcr.io/istio-testing/prowbazel:0.3.5
-          args:
-          - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
-          - "--clean"
-          - "--timeout=60"
-          # Bazel needs privileged mode in order to sandbox builds.
-          securityContext:
-            privileged: true
-          env:
-          - name: GOOGLE_APPLICATION_CREDENTIALS
-            value: /etc/service-account/service-account.json
-          volumeMounts:
-          - name: service
-            mountPath: /etc/service-account
-            readOnly: true
-        volumes:
+  - name: istio-unit-test
+    agent: kubernetes
+    context: prow/istio-unit-test.sh
+    rerun_command: "/test istio-unit-test"
+    trigger: "(?m)^/test (istio-unit-test)?,?(\\s+|$)"
+    spec:
+      containers:
+      - image: gcr.io/istio-testing/prowbazel:0.3.5
+        args:
+        - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
+        - "--clean"
+        - "--timeout=60"
+        # Bazel needs privileged mode in order to sandbox builds.
+        securityContext:
+          privileged: true
+        env:
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /etc/service-account/service-account.json
+        volumeMounts:
         - name: service
-          secret:
-            secretName: service-account
+          mountPath: /etc/service-account
+          readOnly: true
+        # Pilot, security unit tests require kubeconfig
+        - name: kubeconfig
+          mountPath: /home/bootstrap/.kube
+      volumes:
+      - name: service
+        secret:
+          secretName: service-account
+      - name: kubeconfig
+        secret:
+          secretName: istio-e2e-rbac-kubeconfig
+    run_after_success:
     - name: istio-pilot-e2e
```

Comment:
These e2e-kind tests (e2e-smoke, pilot-e2e) should run after istio-persubmit, in your change, these e2e tests will be triggered after istio-unit-test which is the same as we have now. 

So in my idea, it should be like this:

-istio-presubmit (build)
  -run_after_success
    -pilot-e2e
    -e2e-smoke
-istio-unit-test (build and unit test)

Reply:
Done